README.md

```diff
@@ -1,2 +1,178 @@
 # dashboard-template
 A template repository for creating data dashboards with Plotly
+
+## Getting started
+
+### Sign up for GitHub
+
+1.  Sign up for a GitHub account
+1.  Ask the existing team members within the Analytical Dashboards Microsoft Teams to give access to the repository. Make sure to accept the invitation.
```

Comment:
Is this something you'd need to do when getting started?

This feels specific to our team.

Reply:
Derek